src/lib/datadog/index.tsx

```diff
@@ -28,11 +28,11 @@ const datadogScriptBody = `(function(h,o,u,n,d) {
      env: '${ENV}', 
      ${COMMIT_SHA ? `version: '${COMMIT_SHA}',` : ''}
      sampleRate: 100,
-     premiumSampleRate: 100,
+     premiumSampleRate: 0,
+		 sessionReplaySampleRate: 0,
```

Comment:
nit: something off with the indenting on this line?

Reply:
I don't thinks so? looks the same as other lines, at least for me in VSCode 🤔  What are you seeing that looks off?

Reply:
oh it is spaces vs tabs! maybe try copy-pasta the indent space from the previous line?

Reply:
✅ done, copy-pasta'd - let me know if that fixed it!

Reply:
yes that did it! thanks!